src/test/java/org/folio/patronblocks/repository/UserSummaryRepositoryImplTest.java

```diff
@@ -0,0 +1,154 @@
+package org.folio.patronblocks.repository;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import org.awaitility.Awaitility;
+import org.folio.patronblocks.rest.APITests;
+import org.folio.rest.jaxrs.model.UserSummary;
+import org.folio.rest.persist.Criteria.Criterion;
+import org.folio.rest.persist.PostgresClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.vertx.core.Future;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+
+@RunWith(VertxUnitRunner.class)
+public class UserSummaryRepositoryImplTest extends APITests {
+
+  private UserSummaryRepository userSummaryRepository = new UserSummaryRepositoryImpl(
+    PostgresClient.getInstance(vertx, "test_tenant"));
+  private static final String USER_SUMMARY_TABLE = "user_summary";
+
+  @Before
+  public void setUp(TestContext context) {
+    Async async = context.async();
+    PostgresClient.getInstance(vertx, OKAPI_TENANT)
+      .delete(USER_SUMMARY_TABLE, new Criterion(), event -> {
```

Comment:
We're getting two instances of PostgresClient - one time for "test_tenant" and another for OKAPI_TENANT. Is this justified?